forest/src/daemon.rs

```diff
@@ -28,6 +35,32 @@ use wallet::{KeyStore, PersistentKeyStore};
 // TODO benchmark and/or add this as a config option. (1 is temporary value to avoid overlap)
 const WORKER_TASKS: usize = 1;
 
+/// Import a chain from a CAR file
+async fn import_chain<V: ProofVerifier, R: Read, DB: BlockStore>(
+    bs: Arc<DB>,
+    reader: R,
+    snapshot: bool,
+) -> Result<(), Box<dyn std::error::Error>> {
+    println!("Importing chain from snapshot");
+    //start import
+    let cids = load_car(bs.as_ref(), reader)?;
+    let ts = chain::tipset_from_keys(bs.as_ref(), &TipsetKeys::new(cids))?;
+    let gb = chain::tipset_by_height(bs.as_ref(), 0, &ts, true)?.unwrap();
+    let sm = StateManager::new(bs.clone());
+    if !snapshot {
+        info!("Validating imported chain");
+        sm.validate_chain::<V>(ts.clone()).await?;
+    }
+    let gen_cid = chain::set_genesis(bs.as_ref(), &gb.blocks()[0])?;
+    bs.write(chain::HEAD_KEY, ts.key().marshal_cbor()?)?;
```

Comment:
this is done automatically in `set_genesis`, why duplicate?

Reply:
this wasn't addressed

Reply:
Would it be preferred to call `set_heaviest_tipset` which is already public than to make HEAD_KEY public?

Reply:
I'm not sure this will load correctly to use as the correct head, or have you tested?

Reply:
@dutterbutter No, i would have to instantiate a ChainStore to do that because set_heaviest_tipset will update some cache stuff in the ChainStore. 

@austinabell Not sure why you think the head will be wrong? I have ran Forest with a snapshot and got the correct head over RPC.